slurm.conf

```diff
@@ -23,7 +23,7 @@ SlurmctldPidFile=/var/run/slurmd/slurmctld.pid
 SlurmdPidFile=/var/run/slurmd/slurmd.pid
 ProctrackType=proctrack/linuxproc
 #PluginDir=
-CacheGroups=0
+#CacheGroups=0
```

Comment:
Why the changes to slurm.conf and slurmdbd.conf?

Reply:
CacheGroups is gone from newer versions of Slurm, so they refuse to start up if you specify it. Since 0 is the default value, I just commented it out. Same thing for the other changes.